apps/playground/app/test-data-list/page.tsx

```diff
@@ -0,0 +1,242 @@
+                      Active
+                    </Badge>
+                  </DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Name</DataListLabel>
+                  <DataListData>Jane Roe</DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Email</DataListLabel>
+                  <DataListData>janeroe@foo-corpcom</DataListData>
+                </DataListItem>
+
+                <DataListItem align="end">
+                  <DataListLabel>Authentication</DataListLabel>
+                  <DataListData>
+                    <Flex gap="2" mx="-1">
+                      <StarIcon />
+                      <MagicWandIcon />
+                      <StarFilledIcon />
+                    </Flex>
+                  </DataListData>
+                </DataListItem>
+              </DataListRoot>
+            </Flex>
+            <Box mb="6">
+              <Heading mb="5" size="3">
+                With varied content
+              </Heading>
+              <DataListRoot direction={{ initial: 'column', sm: 'row' }} mt="9">
+                <DataListItem>
+                  <DataListLabel>Status!!!!!!!</DataListLabel>
+                  <DataListData>
+                    <Badge color="green" size="1" style={{ marginLeft: -2 }}>
+                      Active
+                    </Badge>
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem align="center">
+                  <DataListLabel>Name</DataListLabel>
+                  <DataListData>
+                    <Button size="1">Add</Button>
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Email</DataListLabel>
+                  <DataListData>vlad@workos.com</DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Organization</DataListLabel>
+                  <DataListData>
+                    <Link href="https://workos.com">WorkOS</Link>
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Long value</DataListLabel>
+                  <DataListData>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla ac nisl et libero
+                    ultricies viverra quis vitae quam. Proin a feugiat metus.
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem align="center">
+                  <DataListLabel>Authentication methods</DataListLabel>
+                  <DataListData>
+                    <Flex gap="2">
+                      <StarFilledIcon />
+                      <StarFilledIcon />
+                    </Flex>
+                  </DataListData>
+                </DataListItem>
```

Comment:
Looks like `align="center"` doesn't quite work in the case when elements in `DataListData` have smaller height than the line-height of the label.

When we implement it for real, let's make sure it works. Also we need a similar visual example for `align="center"`, but when `DataListData` elements are larger in height (which was the original test here).